pcs/common/types.py

```diff
@@ -67,3 +67,10 @@ def from_str(cls, transport: str) -> "CorosyncTransportType":
             return cls(transport.upper())
         except ValueError:
             raise UnknownCorosyncTransportTypeException(transport) from None
+
+
+class CorosyncNodeAddressType(AutoNameEnum):
```

Comment:
In case we want to deserialise this, it should be added to dacite config in `pcs.common.interface.dto.from_dict`